firefox_ui_tests/remote/security/test_unknown_issuer.py

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette import By
+from marionette.errors import MarionetteException
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestUnknownIssuer(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.test_data = 'https://ssl-unknownissuer.mozqa.com'
+
+    def tearDown(self):
+        try:
+            self.windows.close_all([self.browser])
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    def test_unknown_issuer(self):
+        self.marionette.set_context('content')
+
+        # Go to a site that has a cert with an unknown issuer
+        self.assertRaises(MarionetteException, self.marionette.navigate, self.test_data)
+
+        # Wait for about:error page
+        time.sleep(1)
```

Comment:
This comment should be `Wait for the DOM to receive events`. Maybe @chmanchester could also have a look at this given that we might want to have this in Marionette proper?